app/views/health/careplans/_current_goals.haml

```diff
@@ -4,17 +4,7 @@
   - if @goals.present?
     .c-card__content
       - @goals.each_with_index do |goal, index|
-        = render goal, readonly: true, index: (index + 1)
+        = render goal, goal: goal, readonly: true, index: (index + 1)
```

Comment:
is this just so you can reference `goal` instead of whatever the value would be called based the type of goal?  I wonder if we should just replace `render goal` with `render 'goal'...`   Anyway, no harm in what you have.

Reply:
Yes. It made re-using the existing view easier.